geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert2/transforms/IdFunctionFactory.scala

```diff
@@ -70,10 +72,22 @@ class IdFunctionFactory extends TransformerFunctionFactory with LazyLogging {
       hasher.hashString(args(0).toString, StandardCharsets.UTF_8)
   }
 
-  private val murmur3_128: TransformerFunction =
-    new NamedTransformerFunction(Seq("murmur3_128", "murmur3_64"), pure = true) {
-      private val hasher = Hashing.murmur3_128()
-      override def apply(args: Array[AnyRef]): AnyRef =
-        Long.box(hasher.hashString(args(0).toString, StandardCharsets.UTF_8).asLong())
+  private val murmur3_64 =
+    TransformerFunction.pure("murmur3_128", "murmur3_64") { args =>
```

Comment:
Should we add some local comments about the historical reasons for the name being what it is?